### ISSUE TITLE
We are unable to process your request right now. Try again or ask a different question.

### ISSUE THREAD

**Issue by sergeyampo** [gitbito/bitoai#7]
Several days ago request having more than a couple of sentence worked. Now it doesn't. What happened, I can't generate any code and use Bito AI at all.

**Comment by bitomukesh** [comment]
Hi Sergey, thanks for reporting this issue. Recently, we have seen more timeouts from the OpenAI backend and are increasing our capacity. We are also adding support for long input soon.

**Comment by bitoandy** [comment]
@sergeyampo , we have increased our capacity now, so let us know if you are still seeing issues generating code.

There are some limits on input size (but the input length which was working before will work). We have also increased the input size by a bit. instead of 2000 [token](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them), we now support 3000 [tokens](https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them).
